demos/browser/package-lock.json

```diff
@@ -274,6 +274,7 @@
       "requires": {
         "@types/dom-mediacapture-record": "^1.0.4",
         "detect-browser": "^5.1.0",
+        "long": "^4.0.0",
```

Comment:
Is this intentional?

Reply:
Generated by the install script, not by me.